extensions/amp-form/0.1/amp-form.js

```diff
@@ -292,11 +295,19 @@ export class AmpForm {
       user().warn(TAG, 'Variable substitutions disabled for non-canonical ' +
           'origin submit action: %s', this.form_);
     }
+
+    for (const k in formObject) {
+      formData['formFields[' + k + ']'] = formObject[k][0];
```

Comment:
please wrap in `if (formObject.hasOwnProperty(k))` (it is fine without it for now but safer this way to protect against future changes to `getFormAsObject_`)